protractor/e2e/integretion/second-page.spec.ts

```diff
@@ -55,9 +57,22 @@ describe('test for second page', () => {
         await secondPage.waitForElementVisible(secondPage.pdfContent);
     });
 
-    fit('info card error case', async () => {
+    it('info card error case', async () => {
         await secondPage.scrollToElement(secondPage.infoCardSection);
         await secondPage.waitForElementVisible(secondPage.infoEmailInpt);
         await secondPage.infoSubmitBtn.click();
     });
+
+    fit('should open page blog pages with articles', async () => {
+        await secondPage.waitForElementVisible(secondPage.blogHeaderLink);
+        await secondPage.blogHeaderLink.click();
+        await blogPage.waitForElementVisible(blogPage.postClass.first());
+        expect(await blogPage.getCurrentUrl()).toContain(blogPage.pageUrl);
+        expect(await blogPage.postClass.count()).toEqual(20);
+        await blogPage.scrollToElement(blogPage.paginationBlock);
```

Comment:
And user see navigation on the bottom: "First Page 1 2  NextLast Page" - this is your task
Need to cover it like this:
expect(await $('.paging-first').isDisplayed()).toBeTruthy();
        expect(await $('.paging-last').isDisplayed()).toBeTruthy();
        expect(await $('.paging-next').isDisplayed()).toBeTruthy();
        expect(await $$('li .page-numbers').count()).toBe(2);

Reply:
Pagination comment in most libs are pretty standard, I think we don't need additional asserts if the component is visible itself. 
First, Last, Next btn cases should be covered by separate tests